### DISCUSSION TITLE
Can`t copy to RPI connected HDD over Lan anymore Solved

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#932] [Fixed Issues]
Since last 2 updates it is not possible to copy my soundfiles to the HDD , which is connected to the RPI over USB.
Copying from this HDD to my PC is possible . HDD seems to be write protect .

**Comment by hjfisch** [comment]
Did some further testing and after scanning a directory it is write protect. Writing in the Main-Directory is still possible, but then, after scanning, the written new directories are write-protected. 
Tried both of my Systems with Volumio now and no probs .So it must be dedicated to rAudio , because files and directories
are not write protected .
B.t.w Filesystem on HDD is declared as NTFS3 and I had no probs with it untill now .

**Comment by hjfisch** [comment]
I reformat one of my two HDD`s to ext 4 now . To copy my sound will take half a day . So I will report later if the ntfs driver is the problem.

**Comment by hjfisch** [comment]
Not able to copy on ext 4 formated HDD too.
Main directory works , but no subdirectory
Will hopefully work again with a new update soon.

**Comment by hjfisch** [comment]
Maybe it`s a SMB problem , because I could not establish a SMB-Connection from my Linux PC ( all Directories are empty), only works with Windows .
Samba badly needs to be fixed .It`s very slow and needs to be reactivated all the time .

**Reply by clriis** [reply]
I have been running a Pi NAS with USB HDD SMB connect for 6+ years: and have never seen the problems you are describing as long as SAMBA is updated to latest version on host and clients.

My current setup is Pi4 with RaspberryOS 64 bit and two USB3 connected HDDs. I connect from Windows 10, Linux Mint, Ubuntu and Arch Linux.

It is true that there previously were problems with Linux clients not being able to connect to the latest versions of Samba but these problems are solved now.

You should take an extra look at your file permissions and smb.conf file.

**Comment by rern** [comment]
@hjfisch,

Please try:
```sh
sed -i 's/\(force user = \).*/\1http/' /etc/samba/smb.conf
systemctl try-restart smb
```

**Comment by hjfisch** [comment]
Only probs with raudio and Samba since one of the latest updates.

**Comment by hjfisch** [comment]
It`s really weird .I can make new directories in the main folder of the HDD and in Subfolders of the first Subfolder , but not in the first Subfolder . It`s the same on all 3 rAudio Players I have . Tested on some other Audioplayers where it works .
Don`t know what I`m doing wrong .

**Comment by hjfisch** [comment]
Problem solved .
I do not know why , but all first subfolders where write protected .
Loggend in with putty ,  did a chmod and now it works .

**Comment by R101G** [comment]
Probably irrelevant, but are you creating the folders with a file manager? I ask because the manager I use has an option to apply the same permissions to subfolders as to the folder. If that option isn't selected, it can cause a similar problems.

**Comment by hjfisch** [comment]
I do not know , but it happends from one day to another to my media player build in HDD`s formated in NTFS . Than I take them out ,  format to Ext 4 and copied my soundfiles . Maybe I did not look about write protection and so it happends again .
